pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "perceval"
+version = "0.17.1"
+description = "Fetch data from software repositories"
+authors = [
+    "GrimoireLab Developers"
+]
+license = "GPL-3.0+"
+
+readme = "README.md"
+
+homepage = "https://chaoss.github.io/grimoirelab/"
+repository = "https://github.com/chaoss/grimoirelab-perceval"
+
+keywords = [
+    "development",
+    "grimoirelab"
+]
+
+packages = [
+    { include = "perceval" },
+    { include = "tests", format = "sdist" },
+]
+
+classifiers = [
+   "Development Status :: 4 - Beta",
+   "Intended Audience :: Developers",
+   "Topic :: Software Development",
+   "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+   "Programming Language :: Python :: 3"
+]
+
+[tool.poetry.scripts]
+perceval = 'bin.perceval:main'
+
+[tool.poetry.dependencies]
+python = "^3.6"
+python-dateutil = ">=2.6.0"
+requests = ">=2.7.0"
+beautifulsoup4 = ">=4.3.2"
+feedparser = ">=5.1.3"
+dulwich = ">=0.20.0"
+urllib3 = ">=1.22"
+grimoirelab-toolkit = {git = "https://github.com/chaoss/grimoirelab-toolkit.git"}
```

Comment:
This should point to the last version of `grimoirelab-toolkit` instead of to the repository.

Reply:
How does it look now?